public/locales/en/purchase.json

```diff
@@ -0,0 +1,19 @@
+{
+  "title": "Beneath the Surface",
```

Comment:
I'm not sure the title would stay the same across the whole website, but if it does, we should keep it in common?
@tombak98 - do you remember if we discussed this?

Reply:
I don't think the page title (like what shows up in the tab on your browser) would stay the same, I think we'd want it to change depending on the page the user is on, so maybe this would change to "Purchase the book" or something like that?